components/Starknet/modules/architecture_and_concepts/pages/Accounts/deploying_new_accounts.adoc

```diff
@@ -1,83 +1,48 @@
 [id="deploying_new_accounts"]
-= Deploying new accounts
+= Deploying a new account
 
-Starknet provides the `deploy_account` transaction in order to deploy new accounts to the
-network.
+You can deploy a new account in the following ways:
 
-== Deploy account transaction
+* Send a `DEPLOY_ACCOUNT` transaction. This method does not require a preexisting account.
+* Include the `deploy` syscall in the account contract, such as by invoking the Universal Deployer Contract (UDC) with an `INVOKE` transaction. This method requires an existing account to send the `INVOKE` transaction.
 
-In order to deploy a new account on StarkNet you need to complete the following steps:
+Upon receiving one of these transactions, the sequencer performs the following steps:
 
-* Decide on the account contract that you want to deploy
-* Compute your would-be account address off-chain
-* Send funds to this address
+. Verifies that the address has funds to pay for the deployment.
+. Executes the constructor with the given arguments.
+. Runs the respective validation function in the contract, as follows:
+** When deploying with the `DEPLOY_ACCOUNT` transaction type, the sequencer executes the `+__validate_deploy__+` function in the contract.
+** When deploying with the `INVOKE` transaction type and a contract that includes the `deploy` syscall, such as the UDC, the sequencer executes the `+__validate__+` function in the contract.
+. Charges fees from the new account address.
+. Sets the account's nonce to `1`.
```

Comment:
This is only true in case the account is deployed with the `DEPLOY_ACCOUNT` tx. If by the UDC, the nonce of the deployed account is 0.

Reply:
Fixed.